server/src/db/migrations/1_initial-migrations.sql

```diff
@@ -1,6 +1,15 @@
-CREATE TABLE student (
-  student_id SERIAL PRIMARY KEY,
+CREATE EXTENSION "uuid-ossp";
+
+CREATE TABLE "user" (
+  user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
For now, using a UUID is ok. Since we will eventually integrate with Google Workspace for logins and identity, we will replace this with whatever ID Google exposes (whether it be their email address or gaia or something else).

Reply:
I'm not sure you want the primary key of the table to be an externally generated id... as you add external login providers, additional columns (unique/indexed as needed) can be added for linking/mapping...

Reply:
I think we would probably want to have that as a separate column in addition to the `user_id`--ideally primary ID columns have the same type and meaning across all tables in the schema.

Reply:
Good point - having our own IDs makes sense.